src/pages/packer/plugins/index.tsx

```diff
@@ -63,7 +66,9 @@ async function getStaticProps(ctx) {
 		})
 
 		// Replace our original navData with our prepared navData
-		staticProps.props.layoutProps.sidebarNavDataLevels[2].menuItems = navData
+		staticProps.props.layoutProps.sidebarNavDataLevels[2].menuItems = [
+			...navData,
+		]
```

Comment:
🙋🏻 Did you see some behavior that required updating this to copy `navData`? Just curious!

Reply:
Ah thanks for catching this 💖 It's some unintentional cruft from getting Packer plugin pages working. Not needed so I'll revert the change here 🔪